pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -202,3 +202,19 @@ func imageID(containerNameOrID string) (string, error) {
 	}
 	return lines[0], nil
 }
+
+// remove duplicates from a slice
+func removeDuplicates(slice []string) []string {
+	result := []string{}
+	if len(slice) == 0 {
+		return result
+	}
+	seen := make(map[string]struct{})
+	for _, s := range slice {
```

Comment:
1) the loop below is not go formatted properly (should be k := ...)
2) since we're rolling our own:
```go
func removeDuplicates(slice []string) []string {
	result := []string{}
	seenKeys := make(map[string]struct{})
	for _, s := range slice {
		if _, seen := seenKeys[s]; !seen {
			result = append(result, s)
			seenKeys[s] = struct{}{}
		}
	}
	return result
}
```
(faster with one loop, simpler, no need to optimize for empty input, we require an argument)